modules/mw.cx.TargetArticle.js

```
@@ -678,7 +680,7 @@ mw.cx.TargetArticle.prototype.getTags = function (hasTooMuchUnmodifiedText) {
 	return tagString;
 };
 
-mw.cx.TargetArticle.prototype.addMdwikiLinks = function (targetLanguage, targetTitle, qid) {
+mw.cx.TargetArticle.prototype.addMdwikiLinks = function (targetLanguage, targetTitle, qid, wd_result) {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)